src/XrdApps/XrdAppsCconfig.cc

```diff
@@ -186,7 +186,8 @@ int main(int argc, char *argv[])
 //
    Name = XrdOucUtils::InstName(Name);
    sprintf(buff,"%s %s@%s", Xeq, Name, Host);
-   Config = new XrdOucStream(&Say, strdup(buff), &myEnv, "");
+   char* streamName = strdup(buff);
+   Config = new XrdOucStream(&Say, streamName, &myEnv, "");
```

Comment:
Actually, looking at the constructor for `XrdOucStream`, it takes the parameter passed here as `streamName` and does a `strdup` on it to store for itself. So we can pass buff here directly instead of duplicating and then freeing it.

Reply:
True! Made the change.